Integrations/integration-ThreatConnect.yml

```diff
@@ -297,6 +298,12 @@ script:
         indicators = tc.indicators()
         filter1 = indicators.add_filter()
         filter1.add_indicator(demisto.args()['indicator'])
+        owners_arr = []
```

Comment:
You should provide a way to override this with argument:
```
owners = ''
        if demisto.get(demisto.args(), 'owners'):
            owners = demisto.args()['owners']
        if len(owners) > 0:

```

Reply:
@meirwah - added